content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,67 @@
+
+The Manufacture route is activated for each product on its product page. To do so, begin by
+navigating to :menuselection:`Manufacturing --> Products --> Products`, then select an existing
+product or create a new one by clicking :guilabel:`New`.
+
+On the product page, select the :guilabel:`Inventory` tab, then enable the :guilabel:`Manufacture`
+checkbox in the :guilabel:`Routes` section. This tells Odoo that the product can be manufactured.
+
+.. image:: configure_manufacturing_product/manufacturing-route.png
+   :align: center
+   :alt: The Manufacturing route on the Inventory tab of a product page.
+
+Configure a Bill of Materials (BoM)
+===================================
+
+Next, a |BOM| must be configured for the product so that Odoo will know how it is manufactured. A
+|BOM| is a list of the components and operations required to manufacture a product.
+
+To create a |BOM| for a specific product, navigate to :menuselection:`Manufacturing --> Products -->
+Products`, then select the product. On the product page, click the :guilabel:`Bill of Materials`
+smart button at the top of the page, then select :guilabel:`New` to configure a new |BOM|.
+
+.. image:: configure_manufacturing_product/bom-smart-button.png
+   :align: center
+   :alt: The Bill of Materials smart button on a product page.
+
+On the |BOM|, the :guilabel:`Product` field will auto-populate with the product. In the
+:guilabel:`Quantity` field, specify the number of units that the BoM produces.
+
+Add a component to the |BOM| by selecting the :guilabel:`Components` tab and clicking :guilabel:`Add
+a line`. Select a component from the :guilabel:`Component` drop-down menu, then enter the quantity
+in the :guilabel:`Quantity` field. Continue adding components on new lines until all components have
+been added.
+
+.. image:: configure_manufacturing_product/components-tab.png
+   :align: center
+   :alt: The Components tab on a bill of materials.
+
+Next, select the :guilabel:`Operations` tab. Click :guilabel:`Add a line` and a :guilabel:`Create
+Operations` pop-up window appears. In the :guilabel:`Operation` field, specify the name of the
+operation being added (e.g. Assemble, Cut, etc.). Select the work center where the operation will be
+carried out from the :guilabel:`Work Center` drop-down menu. Finally, click :guilabel:`Save & Close`
+to finish adding operations or :guilabel:`Save & New` to add more.
+
+.. image:: configure_manufacturing_product/operations-tab.png
+   :align: center
+   :alt: The Operations tab on a bill of materials.
+
+.. admonition:: Learn more
+
+   The section above provides instructions for creating a basic |BOM| that will allow a product to be
+   manufactured in Odoo. However, it is by no means an exhaustive summary of all the options available
+   when configuring a |BOM|. For more information about bills of materials, see the documentation on
+   how to :ref:`create a bill of materials <manufacturing/management/bill-configuration>`.
```

Comment:
It's also possible to link related eLearning tutorials, if you believe they're suitable (:

The convention for linking eLearning tutorials is "Odoo Tutorials: " followed by the lesson name, then the link to the eLearning page.

```suggestion
   how to :ref:`create a bill of materials <manufacturing/management/bill-configuration>`.
   
 .. see also::
 - `Odoo Tutorials: MO and WO Basics <https://www.odoo.com/slides/slide/702/share>`_
 - `Odoo Tutorials: BoM Basics <https://www.odoo.com/slides/slide/2790/share>`_
```